src/command-line/start.js

```diff
@@ -13,6 +13,7 @@ program
 	.option("-B, --bind <ip>", "bind")
 	.option("    --public", "mode")
 	.option("    --private", "mode")
+  .parse(process.argv)
```

Comment:
I'm trying to understand why we need this here. I would assume the bug is happening in `index.js` due to incorrect order of parsing. There's `program.parseOptions` and `program.parse` calls in there.